### ISSUE TITLE
MWE - NestedSamples() argument 'columns'

### ISSUE THREAD

**Issue by xichenamoy** [williamjameshandley/anesthetic#38]
Adding the argument 'columns = [str]' in function NestedSamples() seems doesn't work well to assign new string/names to the column parameters.  

I tried 
sample = NestedSamples(root=xx)
sample.columns = ['theta', 'beta', 'logL', 'nlive', 'weight']
This would work.

**Comment by lukashergt** [comment]
This is a bit dated, now, but I find that renaming works. One needs to watch out that when providing a list the length of the list needs to match the length of the data.

```python
ns = NestedSamples(root="/scratch/Projects/AnestheticPrj/anesthetic/tests/example_data/pc")
ns.columns = ['y0', 'y1', 'y2', 'y3', 'y4', 'y5', 'y6', 'y7', 'y8']
ns.columns.values[1] = 'z1'
```

This will change the column names from 
```python
['x0', 'x1', 'x2', 'x3', 'x4', 'logL', 'logL_birth', 'nlive', 'weight']
```
as read from the header in the example file to 
```python
['y0', 'z1', 'y2', 'y3', 'y4', 'y5', 'y6', 'y7', 'y8']
```

Or did I misunderstand the problem?

**Comment by williamjameshandley** [comment]
I've encountered this issue in the past when loading chains that lack a `.paramnames` file. The code I'd ideally write would be
```python
ns = NestedSamples(root="<file root>", columns=['here', 'are', 'the', 'parameters'])
```
but at the moment the `root` keyword overrides all the others.

>```python
>ns.columns.values[1] = 'z1'
>```

This is a neat trick, although far from obvious for people who are only just starting out with pandas.

**Comment by lukashergt** [comment]
> The code I'd ideally write would be
> 
> ```python
> ns = NestedSamples(root="<file root>", columns=['here', 'are', 'the', 'parameters'])
> ```

I see, so could this solve it?
```diff
             params, tex = reader.paramnames()
+            columns = kwargs.pop('columns', params)
             limits = reader.limits()
             kwargs['label'] = kwargs.get('label', os.path.basename(root))
-            self.__init__(data=samples, columns=params, w=w, logL=logL,
+            self.__init__(data=samples, columns=columns, w=w, logL=logL,
```

**Comment by williamjameshandley** [comment]
That looks like a pretty good solution. Should be a pretty quick PR if you want to submit one (ideally with a test)

**Comment by lukashergt** [comment]
Fixed in #64
